addon-test-support/-private/dom-helpers.js

```diff
@@ -199,7 +199,8 @@ const POWER_SEELCT_MULTIPLE_OPTIONS_SELECTOR = '.ember-power-select-multiple-opt
 const POWER_SEELCT_MULTIPLE_OPTION_SELECTOR = '.ember-power-select-multiple-option';
 const POWER_SEELCT_MULTIPLE_OPTION_DISABLED_CLASS = 'ember-power-select-multiple-option--disabled';
 const POWER_SELECT_MULTIPLE_OPTION_REMOVE_BUTTON_SELECTOR = '.ember-power-select-multiple-remove-btn';
-const powerSelectDropdownIdForTrigger = (trigger) => trigger.attributes['aria-owns'] && `${trigger.attributes['aria-owns'].value}`;
+
+const powerSelectDropdownIdForTrigger = (trigger) => trigger.attributes['aria-controls'] && `${trigger.attributes['aria-controls'].value}`;
```

Comment:
The change here means this is now working with eps v5, but not anymore with v4, right?

If so, that's ok for me, but should be regarded as a breaking change. Doesn't matter match as we are on a beta prerelease version anyway, so have no good way to convey this breaking change by the version number...

Reply:
Yes, same thoughts here. 😔

AFAIK, SemVer treats all alpha/beta bumps as breaking, despite them being kinda lower than patch.